### PR TITLE
Breadcrumbs

### PR DESCRIPTION
**Pull request by ilkergonenc**
breadcrumbs navigation implementation with new router paths

### CODE REVIEW

File: imports/ui/LayoutContainer.jsx [hunk 1]
```diff
@@ -249,7 +249,7 @@ function Menu({ currentHost, isMobile, screenClass, history }) {
 
   const isCurrentPage = (name) => {
     if (name === 'info') {
-      return pathname.substring(0, 5) === '/page';
+      return pathname.substring(0, 5) === '/pages';
```
Comment (eminx) [inline imports/ui/LayoutContainer.jsx]: `... .substring(0, 6) ...`

File: imports/ui/components/ListMenu.jsx [hunk 2]
```diff
@@ -12,10 +12,12 @@ function ListMenu({ list, pathname, children, ...otherProps }) {
...
+          <Link to={currentUser ? `/@${currentUser?.username}${item.value}` : item.value}>
             <CLink as="span">
               <Text fontWeight={pathname === item.value ? 'bold' : 'normal'}>
-                {tc(`menu.${item.menu}.${item.key}`)}
+                {item.key == 'publicProfile'
+                  ? `@${currentUser?.username}`
+                  : tc(`menu.${item.menu}.${item.key}`)}
               </Text>
             </CLink>
           </Link>
```
Comment (eminx) [inline imports/ui/components/ListMenu.jsx:23]: This component is generically used in different places, and I don't know why we use currentUser routes here...?

**Review by eminx — CHANGES_REQUESTED** [review]
Just the substring needs to be fixed, otherwise good to go I think!

File: imports/ui/LayoutContainer.jsx (hunk 1 above)
Comment (ilkergonenc) [reply imports/ui/LayoutContainer.jsx]: ohhh allright

File: imports/ui/components/ListMenu.jsx (hunk 2 above)
Comment (ilkergonenc) [reply imports/ui/components/ListMenu.jsx:23]: after changing paths like /my-profile to  /@username/profile and added breadcrumbs to those page like 
Home / @username / Profile I realized the users doesn't have a link to see their own public profile so I decided to add to their dashboard menu a direct link .
ListMenu component is used in all dashboard pages and only on dashboard pages both user related and admin related, and they are generated by some constants on utils/constants/general.js ..

Comment (ilkergonenc) [reply imports/ui/components/ListMenu.jsx:23]: maybe it was unnecessary ? not sure

Comment (eminx) [reply imports/ui/components/ListMenu.jsx:23]: All good! thanks for the explanation..

**Review by eminx — APPROVED** [review]

**Pull request merged**